packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
+  .withInitialValues({ transform: [{ translateY: 420 }] })
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
+
+- `.duration(duration: number)` sets length of the animation (in milliseconds). Defaults to '300'.
+- `.delay(durationMs: number)` is the delay before the animation starts (in milliseconds). Defaults to `0`.
+- `.reduceMotion(reduceMotion: ReduceMotion)` determines how the animation responds to the device's reduced motion accessibility setting.
+- `.withInitialValues(values: StyleProps)` allows to override the initial config of the animation.
+- `.withCallback(callback: (finished: boolean) => void)` is the callback that will fire after the animation ends. Sets `finished` to `true` when animation ends without interruptions, and `false` otherwise.
+
+### Remarks
 
-Different type of layout transitions can be customized differently. For the complete list of option please refer to the paragraph specific to the particular animation type. We recommend using builders outside of components or with `useMemo` to ensure the best performance.
+- Just as in [withSpring](/docs/animations/withSpring/) config, spring-based modifiers can be physics-based (has `mass` and `damping` modifiers) or duration-based (has `duration` and `dampingRatio` modifiers). You need to decide whether you want to use physics or duration based animation.
+
+### Example
+
+<ExampleVideo
+  sources={{
+    android:
+      '/react-native-reanimated/recordings/layout_transitions/android_linear_transition.mov',
+    ios: '/react-native-reanimated/recordings/layout_transitions/ios_linear_transition.mov',
+  }}
+/>
+
+### Platform compatibility
+
+<div className="platform-compatibility">
+
+| Android | iOS | Web |
+| ------- | --- | --- |
+| ✅      | ✅  | ✅  |
+
+</div>
+
+## Sequenced Transition
+
+Sequenced transition animates first x-position and width, then later y-position and height.
+
+### Reference
 
 ```js
-const transition = Transition.duration(3000).otherModifier();
+import { SequencedTransition } from 'react-native-reanimated';
 
 function App() {
-  return <AnimatedComponent layout={transition} />;
+  return <Animated.View layout={SequencedTransition} />;
 }
 ```
 
-## Predefined Transitions
+### Modifiers
 
-Below we listed all of the currently available predefined layout transitions. Each transition contains all of its modifiers and a video presenting what it looks like when applied to a waterfall grid.
+```javascript
+SequencedTransition.duration(1000)
+  .delay(500)
+  .reverse()
+  .reduceMotion(ReduceMotion.Never)
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
 
-If you cannot find an transition that suits you then you can create a custom one. If you think that the animation should be here, please open an issue or create a pull request.
+- `.duration(duration: number)` sets length of the animation (in milliseconds). Defaults to '500'.
```

Comment:
This is too slow. Perhaps we will speed it up and unify it to have 300ms everywhere.